### ISSUE TITLE
Sentinl Plugin for ES 6.7.1

### ISSUE THREAD

**Issue by venusf1ytrap** [sirensolutions/sentinl#713]
Recently ES 6.7.1 was launched & sentinl only supports till 6.6 afaik, Any idea by when Sentinl for 6.7.1 will be out? Eagerly awaiting this. Thanks

**Comment by chathsuom** [comment]
ES 7.0.0 also launched. ^_^

**Comment by venusf1ytrap** [comment]
Still awaiting Sentil 6.7.1 :-/ . Any update by when the new version will be out ?

**Comment by jccq** [comment]
Dear all,  with the launch of Open Distro we feel the needs of the Kibana community are sufficiently served and as such we are focusing Sentinl on the needs of the Siren platform only.

The next expected version on Siren will support 6.7.1 (should be short now) and comes with Sentinl in the form of Siren Alert.

Note that starting 10.2 Siren has a free tier now , (again with alerting among the many other things )

There is also a community (just opened) feel free to ask there also. Http://community.siten.io 

Of course PRs here are still welcome

**Comment by Vensis** [comment]
Should be : https://community.siren.io/

**Comment by nirbhayp** [comment]
> Dear all, with the launch of Open Distro we feel the needs of the Kibana community are sufficiently served and as such we are focusing Sentinl on the needs of the Siren platform only.
> 
> The next expected version on Siren will support 6.7.1 (should be short now) and comes with Sentinl in the form of Siren Alert.
> 
> Note that starting 10.2 Siren has a free tier now , (again with alerting among the many other things )
> 
> There is also a community (just opened) feel free to ask there also. [Http://community.siten.io](Http://community.siten.io)
> 
> Of course PRs here are still welcome

Hi @jccq - Just to clarify, does this imply that there are going to be no more releases for the Kibana plugin going forward ?

**Comment by jccq** [comment]
@nirbhayp i wouldn't rule other releases out. They're really just not a priority. We will be reviewing and integrating PRs however and if it all works make releases.

**Comment by fbaligand** [comment]
So if I understand well, @siren won’t continue to bring themselves new features, but only manage external PRs ?

**Comment by jccq** [comment]
Hi @fbaligand , to clarify its not like we are doing this because we dont want the new features to be here.

it is just that we cannot plan regular time to do the porting. Might happen occasionally thought e.g. . No evil.

**Comment by venusf1ytrap** [comment]
@jccq - Is there any plan of releasing Sentilnl for 6.7.1 anytime soon ?

**Comment by fbaligand** [comment]
So @siren team will now concentrate its work on alerting/reporting firstly on opendistro ?

**Comment by lmangani** [comment]
My 2 cents as original creator of Kaae/Sentinl - I think It's time for the #SENTINL community to step up and for some external contributors to actively join to help us keep this project competitive and fresh. I think we've delivered a pretty good baseline for quite some time against all odds keeping this completely open thanks to Siren sponsorship, next hopefully we'll see some of our largest or most affectionate users come forward with concrete help to keep the project healthy and the releases available for general purpose in parallel to the Siren platform evolution, rather than simply expect or even demand releases as if this was a service - remeber it is not! this was born and remains an open-source community project and we need to find that spirit back - it will pay off greatly, for everyone! Spread the word ;)
